server/src/main/java/io/spine/server/entity/storage/EntityColumn.java

```diff
@@ -340,6 +363,51 @@ public Class getType() {
         return getter.getReturnType();
     }
 
+    /**
+     * Returns the type under which the column values are persisted in the data storage.
+     *
+     * <p>For the non-{@link Enumerated} entity columns
+     * this type will be equal to the one retrieved via the {@link #getType()}.
+     *
+     * <p>For {@link Enumerated} columns, see {@link
+     * EnumPersistenceTypes}.
+     *
+     * @return the persistence type of the column values
+     */
+    public Class getPersistedType() {
+        return valuePersistor.getPersistedType();
+    }
+
+    /**
+     * Converts the column value into the value for persistence in the data storage.
+     *
+     * <p>This method can be used to transform the value obtained through the {@link EntityColumn}
+     * getter into the corresponding value used for persistence in the data storage.
+     *
+     * <p>The value type should be the same as the one obtained through the {@link #getType()}. The
+     * output value type will be the same as {@link #getPersistedType()}.
+     *
+     * <p>For the {@code null} input argument the method will always return {@code null},
+     * independently of the declared types.
+     *
+     * <p>The method is accessible outside of the {@link EntityColumn} class to enable the proper
+     * {@link io.spine.client.ColumnFilter} conversion for the {@link
+     * Enumerated} column values.
+     *
+     * @param columnValue the column value to convert
+     * @return the column value converted to the form used for persistence in the data storage
+     * @throws IllegalArgumentException if the value type is not equal to the {@linkplain #getType()
+     *                                  entity column type}
+     */
+    @Nullable
+    public Serializable toPersistedValue(@Nullable Object columnValue) {
```

Comment:
We need to additionally explain why the argument may be `null` and why it makes sense to return `null` in this case back as well.